organizations/organizations.go

```diff
@@ -539,17 +593,25 @@ func (oh *OrganizationHandler) UpgradeToPro(w http.ResponseWriter, r *http.Reque
 }
 
 func IsProVersion(orgID string) (bool, error) {
-	OrgIDFromHex, err := primitive.ObjectIDFromHex(orgID)
-	if err != nil {
-		return false, err
-	}
 
-	organization, err := FetchOrganization(bson.M{"_id": OrgIDFromHex})
-	if err != nil {
-		return false, err
+	if strings.Contains(orgID, "-org") {
+		organization, err := FetchOrganization(bson.M{"_id": orgID})
+		if err != nil {
+			return false, err
+		}
+		return organization.Version == ProVersion, nil
+	} else {
+		OrgIDFromHex, err := primitive.ObjectIDFromHex(orgID)
+		if err != nil {
+			return false, err
+		}
+		organization, err := FetchOrganization(bson.M{"_id": OrgIDFromHex})
+		if err != nil {
+			return false, err
+		}
+		return organization.Version == ProVersion, nil
 	}
 
-	return organization.Version == ProVersion, nil
```

Comment:
Lets revert this, so we have just one occurrence instead of two in the conditions above. It'll be the last thing that runs in this function.